lsp/server/aws-lsp-codewhisperer/src/client/codewhisperer.ts

```diff
@@ -0,0 +1,13 @@
+import { Service } from 'aws-sdk'
+import { ServiceConfigurationOptions } from 'aws-sdk/lib/service'
+const apiConfig = require('./service-2.json')
+import CodeWhispererClient = require('./codewhispererclient')
+
+export function createCodeWhispererClient(options: ServiceConfigurationOptions): CodeWhispererClient {
+    return createService(options) as CodeWhispererClient
+}
+
+function createService(options: ServiceConfigurationOptions): Service {
+    const client = new Service({ apiConfig, ...options } as any)
```

Comment:
I tried a variety of ways to instantiate this service client, and this was the only way I could get to work. I suspect this code may get some updates in the near future (and maybe it can work with v3 of the AWS SDK instead of v2).